### ISSUE TITLE
Reference search for proxyResolver.ts' connectProxyResolver misses results

### ISSUE THREAD

**Issue by chrmarti** [microsoft/vscode#143879]
Testing #143575

On insiders.vscode.dev with the VS Code repo open (full index downloaded):
<img width="904" alt="image" src="https://user-images.githubusercontent.com/9205389/155513081-1b5ddf56-afcc-4bde-9d0f-ca4e710e0ada.png">

Not sure if that is expected. Locally I get:
<img width="421" alt="image" src="https://user-images.githubusercontent.com/9205389/155513244-f7605664-e6a4-4d24-bf1c-a21bae9b9634.png">

**Comment by jrieken** [comment]
This is actually due to TypeScript, not anycode. By default anycode is configured to _not_ offer reference search for TypeScript files. What you see is tsserver lacking cross file support for the web-case. Tho, because it works well for the single file case we didn't enable it by default for anycode. You use the following config to do

```json
	"[typescript]": {
		"anycode.language.features": {
			"completions": false,
			"definitions": false,
			"references": true,
			"highlights": false,
			"outline": false,
			"workspaceSymbols": true,
			"diagnostics": false
		}
	},
```

**Comment by jrieken** [comment]
With the setting I see a call-reference but not the import-reference. I will push a corresponding change

<img width="356" alt="Screen Shot 2022-02-28 at 13 29 45" src="https://user-images.githubusercontent.com/1794099/155983778-969cfbd2-7da8-408d-b8cf-6f19da34c879.png">

**Comment by jrieken** [comment]
Published with 0.0.58
